packages/cli/src/commands/create-app/createApp.ts

```diff
@@ -126,6 +154,15 @@ export default async () => {
     Task.section('Moving to final location');
     await moveApp(tempDir, appDir, answers.name);
 
+    // e2e testing needs special treatment
+    if (process.env.BACKSTAGE_E2E_CLI_TEST) {
+      Task.section('Linking packages locally for e2e tests');
+      const rootDir = process.env.CI
+        ? resolvePath(process.env.GITHUB_WORKSPACE!)
+        : resolvePath(__dirname, '..', '..', '..');
```

Comment:
This is terrible I know...

Reply:
Let's refactor this to add it as `ownRoot` in paths after this is merged.